validator/java/pom.xml

```diff
@@ -557,7 +557,7 @@
         <dependency>
             <groupId>com.yahoo.tagchowder</groupId>
             <artifactId>tagchowder.core</artifactId>
-            <version>2.0.10</version>
+            <version>2.0.17</version>
```

Comment:
Thanks for the upgrade. Could we also fix this security vulnerability?

![image](https://user-images.githubusercontent.com/26553114/79614967-a13df800-80cf-11ea-94c6-14d960d99a5a.png)

Reply:
alright. There was some conflict with maven checkstyles, seems to be resolved, lets see if this builds.

Reply:
whoops ignore this last commit. TIL 8.8 < 8.18